spec/order_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'order'
+
+describe Order do 
+
+  let (:menu){double :menu, dishes: {biriyani: 5.00, uk_curry: 6.00, cheeseburger: 4.00, pizza: 5.00, side_of_salmonella: 1.95}}
```

Comment:
Line is too long. [128/80]<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.